splash/render_options.py

```diff
@@ -97,7 +98,7 @@ def get_wait(self):
         return self.get("wait", defaults.WAIT_TIME, type=float, range=(0, defaults.MAX_WAIT_TIME))
 
     def get_timeout(self):
-        return self.get("timeout", defaults.TIMEOUT, type=float, range=(0, defaults.MAX_TIMEOUT))
+        return self.get("timeout", defaults.TIMEOUT, type=float, range=(0, self.get("max-timeout", defaults.MAX_TIMEOUT, type=float)))
```

Comment:
I think it shouldn't be possible to override max_timeout via GET or POST request. It should be an option set via command-line. It is there to limit resource usage, and allowing to set it using GET/POST doesn't make sense.